indra/databases/obo_client.py

```diff
@@ -3,45 +3,40 @@
 import json
 import logging
 import os
+import pathlib
 import pickle
 import re
 from collections import Counter, defaultdict
+from typing import Optional, Union
 
 import obonet
 
 __all__ = [
+    'OntologyClient',
     'OboClient',
     'RESOURCES',
 ]
 
-HERE = os.path.dirname(os.path.abspath(__file__))
-RESOURCES = os.path.join(HERE, os.pardir, 'resources')
+HERE = pathlib.Path(__file__).parent.resolve()
+RESOURCES = HERE.parent.joinpath('resources').resolve()
```

Comment:
We actually have a function for getting paths to resource files now, see https://indra.readthedocs.io/en/latest/modules/resources.html, so if you're changing this bit of code anyway, might as well use those.

Reply:
Done in 06f333904